sync_diff_inspector/diff.go

```diff
@@ -404,13 +409,16 @@ func (df *Diff) WriteSqls() {
 	}
 }
 
-func generateDML(tp string, data map[string][]byte, keys []*model.ColumnInfo, table *model.TableInfo, schema string) (sql string) {
-	// TODO: can't distinguish NULL between ""
+func generateDML(tp string, data map[string][]byte, null map[string]bool, keys []*model.ColumnInfo, table *model.TableInfo, schema string) (sql string) {
 	switch tp {
 	case "replace":
 		colNames := make([]string, 0, len(table.Columns))
 		values := make([]string, 0, len(table.Columns))
 		for _, col := range table.Columns {
+			// don't need add column name and value for NULL
+			if null[col.Name.O] {
```

Comment:
Should we set `NULL` value explicitly, not just omit it.

Reply:
done